<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>
<title>exp.md</title><link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,700italic,700,400&subset=latin,latin-ext' rel='stylesheet' type='text/css'><style type='text/css'>html, body {overflow-x: initial !important;}html { font-size: 14px; background-color: rgb(255, 255, 255); color: rgb(51, 51, 51); }
body { margin: 0px; padding: 0px; height: auto; bottom: 0px; top: 0px; left: 0px; right: 0px; font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; font-size: 1rem; line-height: 1.42857; overflow-x: hidden; background: inherit; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { background: rgb(181, 214, 252); text-shadow: none; }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; padding-bottom: 70px; white-space: pre-wrap; overflow-x: auto; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
@media screen and (max-width: 500px) {
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
.typora-export #write { margin: 0px auto; }
#write > p:first-child, #write > ul:first-child, #write > ol:first-child, #write > pre:first-child, #write > blockquote:first-child, #write > div:first-child, #write > table:first-child { margin-top: 30px; }
#write li > table:first-child { margin-top: -20px; }
img { max-width: 100%; vertical-align: middle; }
input, button, select, textarea { color: inherit; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; font-size: inherit; line-height: inherit; font-family: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
::before, ::after, * { box-sizing: border-box; }
#write p, #write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write div, #write pre { width: inherit; }
#write p, #write h1, #write h2, #write h3, #write h4, #write h5, #write h6 { position: relative; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
p { -webkit-margin-before: 1rem; -webkit-margin-after: 1rem; -webkit-margin-start: 0px; -webkit-margin-end: 0px; }
.mathjax-block { margin-top: 0px; margin-bottom: 0px; -webkit-margin-before: 0rem; -webkit-margin-after: 0rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: bold; font-style: italic; }
a { cursor: pointer; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; margin: 4px 0px 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 80px; }
.CodeMirror-gutters { border-right: 0px; background-color: inherit; }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
pre { white-space: pre-wrap; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background: inherit; position: relative !important; }
.md-diagram-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
.md-fences .CodeMirror.CodeMirror-wrap { top: -1.6em; margin-bottom: -1.6em; }
.md-fences.mock-cm { white-space: pre-wrap; }
.show-fences-line-number .md-fences { padding-left: 0px; }
.show-fences-line-number .md-fences.mock-cm { padding-left: 40px; }
.footnotes { opacity: 0.8; font-size: 0.9rem; padding-top: 1em; padding-bottom: 1em; }
.footnotes + .footnotes { margin-top: -1em; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; background: transparent; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; -webkit-tap-highlight-color: transparent; line-height: normal; font-weight: normal; text-align: left; box-sizing: content-box; direction: ltr; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li p, li .mathjax-block { margin: 0.5rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; }
@media print {
  html, body { height: 100%; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  h1, h2, h3, h4, h5, h6 { break-after: avoid-page; orphans: 2; }
  p { orphans: 4; }
  html.blink-to-pdf { font-size: 13px; }
  .typora-export #write { padding-left: 1cm; padding-right: 1cm; }
  .typora-export #write::after { height: 0px; }
  @page { margin: 20mm 0mm; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 2.86rem; white-space: pre-wrap; background: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p .md-image:only-child { display: inline-block; width: 100%; text-align: center; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.mathjax-block { white-space: pre; overflow: hidden; width: 100%; }
p + .mathjax-block { margin-top: -1.143rem; }
.mathjax-block:not(:empty)::after { display: none; }
[contenteditable="true"]:active, [contenteditable="true"]:focus { outline: none; box-shadow: none; }
.task-list { list-style-type: none; }
.task-list-item { position: relative; padding-left: 1em; }
.task-list-item input { position: absolute; top: 0px; left: 0px; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc::after, .md-toc-content::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); text-decoration: none; }
.md-toc-inner:hover { }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: bold; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) {
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
.md-tag { opacity: 0.5; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: monospace; }
code { text-align: left; }
h1 .md-tag, h2 .md-tag, h3 .md-tag, h4 .md-tag, h5 .md-tag, h6 .md-tag { font-weight: initial; opacity: 0.35; }
a.md-print-anchor { border-width: initial !important; border-style: none !important; border-color: initial !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: none !important; background: transparent !important; text-decoration: initial !important; text-shadow: initial !important; }
.md-inline-math .MathJax_SVG .noError { display: none !important; }
.mathjax-block .MathJax_SVG_Display { text-align: center; margin: 1em 0em; position: relative; text-indent: 0px; max-width: none; max-height: none; min-height: 0px; min-width: 100%; width: auto; display: block !important; }
.MathJax_SVG_Display, .md-inline-math .MathJax_SVG_Display { width: auto; margin: inherit; display: inline-block !important; }
.MathJax_SVG .MJX-monospace { font-family: monospace; }
.MathJax_SVG .MJX-sans-serif { font-family: sans-serif; }
.MathJax_SVG { display: inline; font-style: normal; font-weight: normal; line-height: normal; zoom: 90%; text-indent: 0px; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0px; min-height: 0px; border: 0px; padding: 0px; margin: 0px; }
.MathJax_SVG * { transition: none; }


@font-face { font-family: "Open Sans"; font-style: normal; font-weight: normal; src: local("Open Sans Regular"), url("./github/400.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: italic; font-weight: normal; src: local("Open Sans Italic"), url("./github/400i.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: normal; font-weight: bold; src: local("Open Sans Bold"), url("./github/700.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: italic; font-weight: bold; src: local("Open Sans Bold Italic"), url("./github/700i.woff") format("woff"); }
html { font-size: 16px; }
body { font-family: "Open Sans", "Clear Sans", "Helvetica Neue", Helvetica, Arial, sans-serif; color: rgb(51, 51, 51); line-height: 1.6; }
#write { max-width: 860px; margin: 0px auto; padding: 20px 30px 100px; }
#write > ul:first-child, #write > ol:first-child { margin-top: 30px; }
body > :first-child { margin-top: 0px !important; }
body > :last-child { margin-bottom: 0px !important; }
a { color: rgb(65, 131, 196); }
h1, h2, h3, h4, h5, h6 { position: relative; margin-top: 1rem; margin-bottom: 1rem; font-weight: bold; line-height: 1.4; cursor: text; }
h1:hover a.anchor, h2:hover a.anchor, h3:hover a.anchor, h4:hover a.anchor, h5:hover a.anchor, h6:hover a.anchor { text-decoration: none; }
h1 tt, h1 code { font-size: inherit; }
h2 tt, h2 code { font-size: inherit; }
h3 tt, h3 code { font-size: inherit; }
h4 tt, h4 code { font-size: inherit; }
h5 tt, h5 code { font-size: inherit; }
h6 tt, h6 code { font-size: inherit; }
h1 { padding-bottom: 0.3em; font-size: 2.25em; line-height: 1.2; border-bottom: 1px solid rgb(238, 238, 238); }
h2 { padding-bottom: 0.3em; font-size: 1.75em; line-height: 1.225; border-bottom: 1px solid rgb(238, 238, 238); }
h3 { font-size: 1.5em; line-height: 1.43; }
h4 { font-size: 1.25em; }
h5 { font-size: 1em; }
h6 { font-size: 1em; color: rgb(119, 119, 119); }
p, blockquote, ul, ol, dl, table { margin: 0.8em 0px; }
li > ol, li > ul { margin: 0px; }
hr { height: 4px; padding: 0px; margin: 16px 0px; background-color: rgb(231, 231, 231); border-width: 0px 0px 1px; border-style: none none solid; border-top-color: initial; border-right-color: initial; border-left-color: initial; border-image: initial; overflow: hidden; box-sizing: content-box; border-bottom-color: rgb(221, 221, 221); }
body > h2:first-child { margin-top: 0px; padding-top: 0px; }
body > h1:first-child { margin-top: 0px; padding-top: 0px; }
body > h1:first-child + h2 { margin-top: 0px; padding-top: 0px; }
body > h3:first-child, body > h4:first-child, body > h5:first-child, body > h6:first-child { margin-top: 0px; padding-top: 0px; }
a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 { margin-top: 0px; padding-top: 0px; }
h1 p, h2 p, h3 p, h4 p, h5 p, h6 p { margin-top: 0px; }
li p.first { display: inline-block; }
ul, ol { padding-left: 30px; }
ul:first-child, ol:first-child { margin-top: 0px; }
ul:last-child, ol:last-child { margin-bottom: 0px; }
blockquote { border-left: 4px solid rgb(221, 221, 221); padding: 0px 15px; color: rgb(119, 119, 119); }
blockquote blockquote { padding-right: 0px; }
table { padding: 0px; word-break: initial; }
table tr { border-top: 1px solid rgb(204, 204, 204); background-color: white; margin: 0px; padding: 0px; }
table tr:nth-child(2n) { background-color: rgb(248, 248, 248); }
table tr th { font-weight: bold; border: 1px solid rgb(204, 204, 204); text-align: left; margin: 0px; padding: 6px 13px; }
table tr td { border: 1px solid rgb(204, 204, 204); text-align: left; margin: 0px; padding: 6px 13px; }
table tr th:first-child, table tr td:first-child { margin-top: 0px; }
table tr th:last-child, table tr td:last-child { margin-bottom: 0px; }
.CodeMirror-gutters { border-right: 1px solid rgb(221, 221, 221); }
.md-fences, code, tt { border: 1px solid rgb(221, 221, 221); background-color: rgb(248, 248, 248); border-radius: 3px; font-family: Consolas, "Liberation Mono", Courier, monospace; padding: 2px 4px 0px; font-size: 0.9em; }
.md-fences { margin-bottom: 15px; margin-top: 15px; padding: 8px 1em 6px; }
.task-list { padding-left: 0px; }
.task-list-item { padding-left: 32px; }
.task-list-item input { top: 3px; left: 8px; }
@media screen and (min-width: 914px) {
}
@media print {
  html { font-size: 13px; }
  table, pre { break-inside: avoid; }
  pre { word-wrap: break-word; }
}
.md-fences { background-color: rgb(248, 248, 248); }
#write pre.md-meta-block { padding: 1rem; font-size: 85%; line-height: 1.45; background-color: rgb(247, 247, 247); border: 0px; border-radius: 3px; color: rgb(119, 119, 119); margin-top: 0px !important; }
.mathjax-block > .code-tooltip { bottom: 0.375rem; }
#write > h3.md-focus::before { left: -1.5625rem; top: 0.375rem; }
#write > h4.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
#write > h5.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
#write > h6.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
.md-image > .md-meta { border: 1px solid rgb(221, 221, 221); border-radius: 3px; font-family: Consolas, "Liberation Mono", Courier, monospace; padding: 2px 4px 0px; font-size: 0.9em; color: inherit; }
.md-tag { color: inherit; }
.md-toc { margin-top: 20px; padding-bottom: 20px; }
#typora-quick-open { border: 1px solid rgb(221, 221, 221); background-color: rgb(248, 248, 248); }
#typora-quick-open-item { background-color: rgb(250, 250, 250); border-color: rgb(254, 254, 254) rgb(229, 229, 229) rgb(229, 229, 229) rgb(238, 238, 238); border-style: solid; border-width: 1px; }
#md-notification::before { top: 10px; }
.on-focus-mode blockquote { border-left-color: rgba(85, 85, 85, 0.117647); }
header, .context-menu, .megamenu-content, footer { font-family: "Segoe UI", Arial, sans-serif; }






</style>
</head>
<body class='typora-export' >
<div  id='write'  class = 'is-node'><h1><a name='header-c5' class='md-header-anchor '></a>实验一 测序数据的质量控制与预处理</h1><h2><a name='header-c7' class='md-header-anchor '></a>实验目的</h2><ol start='' ><li>掌握测序数据的FASTQ格式。</li><li>熟悉FastQC、FASTX-Toolkit等质量控制工具的使用方法。</li><li>熟悉Galaxy的使用方法。</li><li>了解FastQC输出结果的含义。</li></ol><h2><a name='header-c21' class='md-header-anchor '></a>实验材料</h2><ol start='' ><li><a href='https://github.com/bioinformatics-core-shared-training/cruk-bioinf-sschool/blob/gh-pages/Day1/qa/sample.fastq'>sample.fastq</a></li><li><a href='https://github.com/bioinformatics-core-shared-training/cruk-bioinf-sschool/blob/gh-pages/Day1/qa/sample2.fastq'>sample2.fastq</a></li></ol><h2><a name='header-c29' class='md-header-anchor '></a>实验工具</h2><ol start='' ><li><a href='https://usegalaxy.org/'>Galaxy</a></li><li><a href='http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'>FastQC</a></li><li><a href='http://hannonlab.cshl.edu/fastx_toolkit/'>FASTX-Toolkit</a></li></ol><h2><a name='header-c40' class='md-header-anchor '></a>实验步骤</h2><ol start='' ><li><p>Upload data to Galaxy</p><ul><li><p>工具：&quot;Get data&quot; ---&gt; &quot;Upload File&quot;</p></li><li><p>数据：sample.fastq, sample2.fastq</p></li><li><p>注意：</p><ul><li>既可以通过链接获取数据，也可以直接上传本地数据（推荐前者）</li><li>选择正确的数据格式（提示：Fastq；sample.fastq: Illumina 1.5; sample2.fastq: Illumina 1.5）</li><li>因为基因组版本在本实验中无关紧要，所以随便选择一个即可（比如：hg19）</li></ul></li><li><p>思考：在实际的数据处理中，如何</p><ul><li>获取测序数据</li><li>拿到数据格式（Fastq的质量编码类型）</li><li>选择基因组版本</li></ul></li></ul></li><li><p>Checking read quality with FastQC</p><ul><li><p>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;FastQC&quot;</p></li><li><p>数据：sample.fastq</p></li><li><p>注意：理解输出报告中每一部分结果的含义</p></li><li><p>思考：</p><ul><li>如何查找FastQC的使用说明？</li><li>&quot;Basic Statistics&quot;中的Encoding说明什么？</li><li>从&quot;Per base sequence quality&quot;中能得到什么信息？</li><li>从&quot;Per base sequence content&quot;中能得到什么信息？</li></ul></li></ul></li><li><p>Convert FASTQ quality to sanger</p><ul><li>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;FASTQ Groomer&quot;</li><li>数据：sample.fastq</li><li>注意：指定正确的输入数据的质量编码类型（提示：Illumina 1.5）</li><li>思考：为什么首先要把Fastq的质量编码转换成Sanger，之后才进行后续的处理？</li></ul></li><li><p>Preprocessing with FASTX-Toolkit</p><ol start='' ><li><p>Remove reads with lower quality</p><ul><li><p>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;Filter by quality&quot;</p></li><li><p>数据：sample_sanger.fastq</p></li><li><p>注意：设定正确的参数（要求：keeping only reads that have at least 75% of bases with a quality score of 20 or more）</p></li><li><p>思考：</p><ul><li>总的输入、最终输出、丢掉的reads数目各是多少？</li><li>在实际的数据处理中，如何选择<strong>合适</strong>的参数？</li></ul></li></ul></li><li><p>Trim the bases with sequence bias from reads</p><ul><li><p>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;Trim sequences&quot;</p></li><li><p>数据：sample_sanger_filtered.fastq</p></li><li><p>注意：设定正确的参数</p><ul><li>参考FastQC输出报告中的&quot;Per base sequence content&quot;设定参数&quot;First base to keep&quot;</li><li>参考FastQC输出报告中的&quot;Basic Statistics&quot;或者&quot;Sequence Length Distribution&quot;设定参数&quot;Last base to keep&quot;</li></ul></li><li><p>思考：在实际的数据处理中，如何选择<strong>合适</strong>的参数？</p></li></ul></li></ol></li><li><p>Clean adapter containing reads from FASTQ data</p><ol start='' ><li><p>Checking read quality with FastQC</p><ul><li><p>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;FastQC&quot;</p></li><li><p>数据：sample2.fastq</p></li><li><p>思考：</p><ul><li>&quot;Basic Statistics&quot;中的Encoding说明什么？</li><li>从&quot;Overrepresented sequences&quot;中能得到什么信息？</li></ul></li></ul></li><li><p>Convert FASTQ quality to sanger</p><ul><li>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;FASTQ Groomer&quot;</li><li>数据：sample2.fastq</li><li>注意：指定正确的输入数据的质量编码类型（提示：Illumina 1.5）</li></ul></li><li><p>Clean adapter containing reads</p><ul><li><p>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;Trim Galore!&quot;</p></li><li><p>数据：sample2_sanger.fastq</p></li><li><p>注意：设定正确的参数，要求如下</p><ul><li>Throw away processed reads shorter than 20 bases</li><li>The level of error tolerance is adjusted by specifying a maximum 10% error rate</li></ul></li><li><p>思考：</p><ul><li>如何指定adapter的序列？（提示：FastQC输出报告中的“Overrepresented sequences”）</li><li>总的输入、带有adapter的reads数目各是多少？</li><li>尝试使用&quot;NGS: QC and manipulation&quot; ---&gt; &quot;Clip&quot;去除adapter，并比较两种工具的结果。</li></ul></li></ul></li><li><p>Checking read quality after cleaning adaper</p><ul><li><p>工具：&quot;NGS: QC and manipulation&quot; ---&gt; &quot;FastQC&quot;</p></li><li><p>数据：sample2_sanger_trim.fastq</p></li><li><p>思考：</p><ul><li>比较去除adapter前后的FastQC输出报告。</li><li>不去除adapter的话对后续的处理有没有影响？</li></ul></li></ul></li></ol></li><li><p>探索&quot;NGS: QC and manipulation&quot;中的其他工具</p></li></ol><h2><a name='header-c266' class='md-header-anchor '></a>参考资料</h2><ul><li><a href='http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc/Help/'>FastQC Help</a></li><li><a href='https://github.com/bioinformatics-core-shared-training/cruk-bioinf-sschool/blob/gh-pages/Day1/fastqc_sweave.pdf'>fastqc_sweave.pdf</a></li><li><a href='https://github.com/bioinformatics-core-shared-training/cruk-bioinf-sschool/tree/gh-pages/Day1/qa/results'>QC results</a></li></ul><h2><a name='header-c277' class='md-header-anchor '></a>备注</h2><ul><li>除了Main public site上的<a href='https://usegalaxy.org/'>Galaxy</a>以外，还可以尝试使用其他Public servers，比如<a href='https://bioinf-galaxian.erasmusmc.nl/galaxy/'>Erasmus MC Bioinformatics Galaxy Server</a>等。</li></ul><hr /><p></p><h1><a name='header-c285' class='md-header-anchor '></a>实验二 外显子组测序数据的处理</h1><h2><a name='header-c286' class='md-header-anchor '></a>实验目的</h2><ol start='' ><li>掌握外显子组测序数据的分析流程。</li><li>熟悉BWA、SAMtools、SnpEff等工具的使用方法。</li><li>熟悉Galaxy的使用方法。</li><li>了解存储变异信息的VCF格式。</li></ol><h2><a name='header-c300' class='md-header-anchor '></a>实验材料</h2><ol start='' ><li>NA8524_chr21.fq: human(hg19), fastqsanger</li></ol><h2><a name='header-c305' class='md-header-anchor '></a>实验工具</h2><ol start='' ><li><a href='https://usegalaxy.org/'>Galaxy</a></li><li><a href='http://bio-bwa.sourceforge.net/'>BWA</a></li><li><a href='http://www.htslib.org/'>SAMtools</a></li><li><a href='http://snpeff.sourceforge.net/'>SnpEff</a></li></ol><h2><a name='header-c319' class='md-header-anchor '></a>实验步骤</h2><ol start='' ><li><p>Upload data to Galaxy（略；参看实验一）</p></li><li><p>Checking read quality with FastQC（略；参看实验一）</p></li><li><p>Preprocessing（略；参看实验一）</p></li><li><p>Map with BWA</p><ul><li>工具：&quot;NGS: Mapping&quot; ---&gt; &quot;Map with BWA for Illumina&quot;</li><li>数据：NA8524_chr21.fq</li><li>注意：指定合适的基因组组装版本</li><li>思考：尝试&quot;Map with Bowtie for Illumina&quot;、&quot;Map with BWA&quot;等工具</li></ul></li><li><p>Statistics with SAMtools</p><ol start='' ><li><p>Convert SAM to BAM</p><ul><li>工具：&quot;NGS: SAMtools&quot; ---&gt; &quot;SAM-to-BAM&quot;</li></ul></li><li><p>Print descriptive information for a BAM dataset</p><ul><li>工具：&quot;NGS: SAMtools&quot; ---&gt; &quot;Flagstat&quot;</li><li>思考：尝试&quot;Stats&quot;、&quot;IdxStats&quot;等质控工具</li></ul></li></ol></li><li><p>Call variants</p><ol start='' ><li><p>Call variants with MPileup</p><ul><li><p>工具：&quot;NGS: SAMtools&quot; ---&gt; &quot;MPileup&quot;</p></li><li><p>注意：设定正确的参数</p><ul><li>选择和先前一致的基因组组装版本</li><li>设定参数&quot;Genotype Likelihood Computation&quot;为&quot;Do not perform genotype likelihood computation (output pileup)&quot;</li></ul></li><li><p>思考：尝试直接使用MPileup提取变异（跳过后面的Varscan）</p></li></ul></li><li><p>Variant detection with Varscan</p><ul><li>工具：&quot;NGS: Variant Analysis&quot; ---&gt; &quot;Varscan&quot;</li><li>思考：尝试调整&quot;Varscan&quot;的参数</li></ul></li></ol></li><li><p>Annotate variants</p><ul><li><p>工具：&quot;NGS: Variant Analysis&quot; ---&gt; &quot;SnpEff&quot;</p></li><li><p>注意：设定合适的参数</p></li><li><p>思考：尝试&quot;ANNOVAR Annotate VCF&quot;等其他注释工具</p><ul><li>提示：&quot;ANNOVAR Annotate VCF&quot;可能无法正常使用</li></ul></li></ul></li><li><p>Filter variants</p><ul><li><p>工具：&quot;NGS: VCF Manipulation&quot; ---&gt; &quot;VCFfilter&quot;</p></li><li><p>注意：根据自己的需要构建表达式</p><ul><li>提示：&quot;VCFfilter&quot;可能无法正常使用</li></ul></li><li><p>思考：尝试使用&quot;Text Manipulation&quot;和&quot;Filter and Sort&quot;中的工具处理VCF</p></li></ul></li><li><p>补充：实际的数据处理过程中还需要对比对结果（BAM文件）和变异数据（VCF文件）进行以下处理</p><ul><li>Mark/Remove PCR Duplicates</li><li>Local Realignments Around Indels</li><li>Quality Recalibration</li></ul></li></ol><h2><a name='header-c448' class='md-header-anchor '></a>参考资料</h2><ul><li><a href='https://usegalaxy.org/u/jeremy/w/exome-analysis'>Galaxy Workflow &#39;Exome Analysis&#39;</a></li></ul><h2><a name='header-c453' class='md-header-anchor '></a>备注</h2><ul><li>除了Main public site上的<a href='https://usegalaxy.org/'>Galaxy</a>以外，还可以尝试使用其他Public servers，比如<a href='https://bioinf-galaxian.erasmusmc.nl/galaxy/'>Erasmus MC Bioinformatics Galaxy Server</a>等。</li></ul><hr /><p></p><h1><a name='header-c461' class='md-header-anchor '></a>实验三 RNA-Seq的数据处理</h1><h2><a name='header-c462' class='md-header-anchor '></a>实验目的</h2><ol start='' ><li>掌握RNA-Seq测序数据的分析流程。</li><li>熟悉Tuxedo套件的使用方法。</li><li>熟悉Galaxy的使用方法。</li><li>了解存储注释信息的GTF/GFF格式。</li></ol><h2><a name='header-c476' class='md-header-anchor '></a>实验材料</h2><ol start='' ><li><a href='http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq_files/h1-hESC_Sample_Dataset.fastqsanger'>h1-hESC_Sample_Dataset.fastqsanger</a>: human(hg19), fastqsanger</li><li><a href='http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq_files/GM12878_Sample_Dataset.fastqsanger'>GM12878_Sample_Dataset.fastqsanger</a>: human(hg19), fastqsanger</li><li><a href='http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq_files/UCSC_Main_on_Human__refGene_chr19_BED.bed'>UCSC_Main_on_Human<em>_refGene_chr19</em>BED.bed</a></li><li><a href='http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq_files/UCSC_Main_on_Human__refGene_chr19_GTF.gtf'>UCSC_Main_on_Human<em>_refGene_chr19</em>GTF.gtf</a></li></ol><h2><a name='header-c490' class='md-header-anchor '></a>实验工具</h2><ol start='' ><li><a href='https://usegalaxy.org/'>Galaxy</a></li><li><a href='http://bowtie-bio.sourceforge.net/index.shtml'>Bowtie</a></li><li><a href='https://ccb.jhu.edu/software/tophat/index.shtml'>TopHat</a></li><li><a href='http://cole-trapnell-lab.github.io/cufflinks/'>Cufflinks</a></li></ol><h2><a name='header-c504' class='md-header-anchor '></a>实验步骤</h2><ol start='' ><li><p>Upload data to Galaxy（略；参看实验一）</p></li><li><p>Checking read quality with FastQC（略；参看实验一）</p><ul><li>思考：可以尝试&quot;NGS: QC and manipulation&quot;中的&quot;FastQC&quot;、&quot;Build base quality distribution&quot;、&quot;Draw quality score boxplot&quot;、&quot;Compute quality statistics&quot;、&quot;FASTQ Summary Statistics&quot;（结合&quot;Graph/Display Data&quot;中的&quot;Boxplot&quot;使用）等工具</li></ul></li><li><p>Preprocessing（略；参看实验一）</p><ul><li><p>思考</p><ul><li>标准：remove base positions that have a median quality score of below 15</li><li>Is trimming needed for the datasets?</li><li>If necessary, trim the reads! （可以尝试&quot;NGS: QC and manipulation&quot;中的&quot;Trim Galore!&quot;、&quot;Trimmomatic&quot;、&quot;Trim sequences&quot;、&quot;FASTQ Trimmer&quot;、&quot;FASTQ Quality Trimmer&quot;等工具）</li></ul></li></ul></li><li><p>Map reads with TopHat</p><ul><li><p>工具：&quot;NGS: RNA Analysis&quot; ---&gt; &quot;TopHat&quot;</p></li><li><p>数据：*.fastq</p></li><li><p>注意：指定合适的基因组组装版本</p></li><li><p>思考</p><ul><li>可以尝试利用&quot;UCSC_Main_on_Human<em>_refGene_chr19</em>BED.bed&quot;对TopHat的结果进行可视化</li><li>理解TopHat主要参数的含义</li><li>理解TopHat每个输出文件的含义</li></ul></li></ul></li><li><p>Assemble and analyze transcripts</p><ul><li><p>工具：&quot;NGS: RNA Analysis&quot; ---&gt; &quot;Cufflinks&quot;</p></li><li><p>数据：accepted_hits.bam</p></li><li><p>注意：设置合适的参数（此处默认即可）</p><ul><li>理解Cufflinks主要参数的含义</li><li>理解Cufflinks每个输出文件的含义</li></ul></li></ul></li><li><p>Identify transcripts that are differentially expressed</p><ol start='' ><li><p>Compare assembled transcripts</p><ul><li><p>工具：&quot;NGS: RNA Analysis&quot; ---&gt; &quot;Cuffcompare&quot;</p></li><li><p>数据：(assembled transcripts) X 2 + &quot;UCSC_Main_on_Human<em>_refGene_chr19</em>GTF.gtf&quot;</p></li><li><p>注意：设置合适的参数</p><ul><li>理解Cuffcompare主要参数的含义</li><li>理解Cuffcompare每个输出文件的含义</li></ul></li></ul></li><li><p>Find significant changes in transcript expression</p><ul><li><p>工具：&quot;NGS: RNA Analysis&quot; ---&gt; &quot;Cuffdiff&quot;</p></li><li><p>数据：combined transcripts + (accepted_hits.bam) X 2</p></li><li><p>注意：设置合适的参数</p><ul><li>指定“Condition Name”</li><li>理解Cuffdiff主要参数的含义</li><li>理解Cuffdiff每个输出文件的含义</li></ul></li><li><p>思考：分别从&quot;transcript differential expression testing&quot;和&quot;gene differential expression testing&quot;中提取显著差异表达的转录本和基因</p></li></ul></li></ol></li><li><p>Visualization with CummeRbund（略）</p></li></ol><h2><a name='header-c632' class='md-header-anchor '></a>参考资料</h2><ul><li><a href='http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq.html'>RNA-Seq using Galaxy</a></li><li><a href='http://hpc.ilri.cgiar.org/training/Glossina_annotation/RNA-Seq_files/'>/training/Glossina_annotation/RNA-Seq_files</a></li></ul><h2><a name='header-c640' class='md-header-anchor '></a>备注</h2><ul><li>除了Main public site上的<a href='https://usegalaxy.org/'>Galaxy</a>以外，还可以尝试使用其他Public servers，比如<a href='https://bioinf-galaxian.erasmusmc.nl/galaxy/'>Erasmus MC Bioinformatics Galaxy Server</a>等。</li></ul><hr /><p></p></div>
</body>
</html>